reduceEpsilons.py

```diff
@@ -176,4 +194,4 @@ def main(tolerance, step, username):
         print("{} exceeds maximum step of 0.00001".format(step))
         exit(1)
 
-    main(tolerance, step, username)
\ No newline at end of file
+    main(tolerance, step, username, configuration)
```

Comment:
This will cause an error since the signature of the function wasn't updated and I don't think I caught `configuration` being a global variable.